tests/redisUnit.js

```diff
@@ -0,0 +1,199 @@
+function quitDB(client) {
+  return new Promise((resolve) => {
+    client.flushdb(() => {
+      // never fails
+      resolve();
+    });
+  })
+  .then(() => new Promise((resolve) => {
+    client.quit(() => {
+      // do not check for errors, just resolve
+      resolve();
+    });
+  }));
+}
```

Comment:
I think we can avoid creating two separate `Promise`s here:

    return new Promise((resolve) => {
      // never fails
      client.flushdb(() => {
        client.quit(() => {
          // do not check for errors, just resolve
          resolve();
        });
      });
    });

I don't think there's a difference in semantics; go with whatever looks cleaner